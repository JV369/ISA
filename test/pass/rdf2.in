./feedreader http://www.mocky.io/v2/5bcdea4a2f00002c00c85638 -a -u
