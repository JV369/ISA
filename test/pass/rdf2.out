*** Test RDF ***
Test item RDF
Autor: JV
URL: http://c.moreover.com/click/here.pl?r123

Test item RDF2
Autor: Hue
URL: http://huerdf.com

